docs/data-cookbooks/error-handling.md

```diff
@@ -17,100 +17,222 @@ The former (tasks which throw errors which are not `WorkflowError`s) is the expe
-* States may be configured to `Retry` a task on specified failures to handle transient issues, such as those arising from resource allocation throttling, instead of failing the entire workflow. Cumulus supports the AWS retry configuration outlined [here](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-error-handling.html) and an example is provided in the `HelloWorld` step of the `RetryPassWorkflow` workflow defined in the Cumulus repository's [example workflows `errorsAndRetries.yml`](https://github.com/nasa/cumulus/blob/master/example/workflows/errorsAndRetries.yml).
-* Tasks downstream of failed tasks should understand how to pass along exceptions if required: If a task throws an error which is caught by the workflow configuration and passed to another state which also uses the CMA, the CMA overrides the exception key to `"None"` so the exception will not be passed to downstream tasks after the next state. This is okay if the exception is not needed in downstream tasks. If the exception is needed in downstream tasks, you need to re-attach the exception to the Cumulus message by setting the `ResultPath` to be `$.exception` for the task where the error is initially caught. In the example below, `CnmResponseFail` catches and re-attaches the error to the message.
-  * If multiple downstream tasks should run after a workflow task has thrown an error, you can create a separate failure branch of your workflow by chaining tasks that catch and re-attach the error as described above.
+* States may be configured to `Retry` a task on specified failures to handle transient issues, such as those arising from resource allocation throttling, instead of failing the entire workflow. Cumulus supports the AWS retry configuration outlined [here](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-error-handling.html) and an example is provided in the `HelloWorld` step of the `RetryPassWorkflow` workflow defined in the Cumulus repository's [example workflow `retry_pass_workflow`](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/retry_pass_workflow.tf).
+* Tasks downstream of failed tasks should understand how to pass along exceptions if required.
+  If a task throws an error which is caught by the workflow configuration and passed to another state which also uses the CMA, the CMA overrides the exception key to `"None"` so the exception will not be passed to downstream tasks after the next state. This works if the exception is not needed in downstream tasks. If the exception is needed in downstream tasks, you need to re-attach the exception to the Cumulus message by setting the `ResultPath` to be `$.exception` for the task where the error is initially caught. In the example below, `CnmResponseFail` catches and re-attaches the error to the message.
```

Comment:
Is there supposed to be an asterisk in front of this paragraph? To make it part of the same list as the other items below it